sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -543,6 +543,7 @@ export class ManagementClient extends LinkEntity {
       if (!item.messageId) item.messageId = generate_uuid();
       item.scheduledEnqueueTimeUtc = scheduledEnqueueTimeUtc;
       const amqpMessage = toAmqpMessage(item);
+      amqpMessage.body = this._context.namespace.dataTransformer.encode(amqpMessage.body);
```

Comment:
Is this encoding already happening in other code paths where we send messages and it was just missed here before?

Reply:
Yes.